composer.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "magento/magento2ce",
-    "description": "Magento 2 (Open Source)",
+    "name": "mage-os/magento2ce",
+    "description": "Mage-OS (Open Source)",
```

Comment:
Please change the package name back to `magento/magento2ce` since that is changed by the release build script based on the parameters (it doesn't always have to be the `mage-os` vendor name).

Reply:
This change actually was fixing https://github.com/vpodorozh/mageos-magento2/actions/runs/7146821940/job/19465285198
So together with reverting this, we should adjust test as well.

Reply:
@Vinai - have a look here: https://github.com/vpodorozh/mageos-magento2/pull/10 

I've created a separate PR to work on this, as it raises some questions. Thx

Reply:
As discussed in the call, we keep the `mage-os` vendor name, since it is required for the version meta-data shown in the admin (and other places)